### DISCUSSION TITLE
ImportError: cannot import name 'ReducedRegressionForecaster' from 'sktime.forecasting.compose'

### DISCUSSION THREAD

**Discussion by aiexplorations** [alan-turing-institute/sktime#754] [General]
I get this when doing an import on the Forecasting example iPython notebook.

https://github.com/alan-turing-institute/sktime/blob/main/examples/01_forecasting.ipynb 

Posting this here before the issues list just in case there is a workaround already.

Anyone else has the same error, or found workarounds? Would love to know, thanks.

```
ImportError                               Traceback (most recent call last)
<ipython-input-4-e5215cdd46a9> in <module>
      7 from sktime.forecasting.arima import ARIMA, AutoARIMA
      8 from sktime.forecasting.base import ForecastingHorizon
----> 9 from sktime.forecasting.compose import (
     10     EnsembleForecaster,
     11     ReducedRegressionForecaster,

ImportError: cannot import name 'ReducedRegressionForecaster' from 'sktime.forecasting.compose' (/home/rajesh/anaconda3/lib/python3.8/site-packages/sktime/forecasting/compose/__init__.py)

```

**Comment by mloning** [comment]
We accidentally changed the name to `ReducedForecaster` without deprecating it first. `from sktime.forecasting.compose import ReducedRegressionForecaster` should work.
